config.go

```diff
@@ -269,6 +269,68 @@ func (c *config) discoverExternalComponents(path string) error {
 		externallyUsed = nil
 	}
 
+	pluginPaths, err = c.discoverSingle(filepath.Join(path, "packer-plugin-*"))
+	if err != nil {
+		return err
+	}
+
+	for pluginName, pluginPath := range pluginPaths {
+		if err := c.discoverMultiPlugin(pluginName, pluginPath); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// discoverMultiPlugin takes the description from a multiplugin binary and
+// makes the plugins available to use in Packer. Each plugin found in the
+// binary will be addressable using `${pluginName}-${builderName}` for example.
+// pluginName could be manually set. It usually is a cloud name like amazon.
+// pluginName can be extrapolated from the filename of the binary; so
+// if the "packer-plugin-amazon" binary had an "ebs" builder one could use
+// the "amazon-ebs" builder.
+func (c *config) discoverMultiPlugin(pluginName, pluginPath string) error {
+	out, err := exec.Command(pluginPath, "describe").Output()
+	if err != nil {
+		return err
```

Comment:
This is the place where we could fall back to cloud name-matching, like I suggested in the RFC.

Reply:
@SwampDragons Sorry, I'm not sure I understood your suggestion correctly then 🤦🏼  do you mind explaining it again ? Could be zoom or here ?

Reply:
I'm thinking a bit further here, but is this something we could check for when validating a plugin? 
I mean, in the future we add a check that validates that a multi plugin has the describe command and it returns a valid description.

Reply:
@sylviamoss, yup I think this could be extracted for this: like L293-L300 .